### DISCUSSION TITLE
It would be nice if there is an option to adjust the contrast ratio

### DISCUSSION THREAD

**Discussion by IceCodeNew** [community/community#22211] [Accessibility]
<p>It would be nice if there is an option to adjust the contrast ratio.</p>

**Comment by gaknoll** [comment, marked as answer]
<p>Thanks for the suggestion <a class="mention" href="/icecodenew">@IceCodeNew</a>! We have a couple topics from the community asking for a theme with <a href="https://github.community/t/dark-mode-theme-with-less-contrast/147845/50">less contrast</a> and a theme with <a href="https://github.community/t/a-dark-mode-with-more-contrast/147815/11">more contrast</a>. We’re currently working on high contrast themes for accessibility as well as other themes so dark mode is really just the beginning!</p>

**Comment by cehfisher** [comment]
Hi @IceCodeNew - with all of the [theme preferences](https://github.com/settings/appearance) now built into the GitHub interface including themes with less/more contrast, could we consider this stale issue resolved? If so, could you close this issue? If not, what would be the next steps for this feedback request? (cc @queenofcorgis)
